app/manage-my-booking-info/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import Link from 'next/link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleLeft } from '@fortawesome/free-solid-svg-icons';
+
+export default function ManageBooking() {
+    return (
+        <div className="min-h-screen flex flex-col items-center justify-center bg-gray-100 px-4 sm:px-6 lg:px-8">
+            {/* Logo and Back Button */} 
+            <div className="absolute top-24">
+                <Link href="/" passHref>
+                    <span className="flex items-center text-gray-600 hover:text-gray-800 text-xl font-semibold">
+                        <FontAwesomeIcon icon={faCircleLeft} className="mr-2" />
+                        <h6>Back</h6>
+                    </span>
+                </Link>
+            </div>
+
+            <div className="absolute top-10 left-10 flex items-center">
+                <img src="/logo.png" alt="Logo" className="h-10 w-20 sm:h-12 sm:w-24 lg:h-14 lg:w-28" />
+            </div>
+
+            {/* Form */}
+            <div className="w-full max-w-md sm:max-w-lg lg:max-w-xl bg-white p-6 rounded-lg shadow-md">
+                <h2 className="text-2xl font-bold mb-4 text-center">Manage your reservation</h2>
+
+                <p className="text-sm sm:text-base font-medium mb-6 text-gray-600">Change your booking as you wish, quickly and easily.</p>
+
+                <form>
```

Comment:
Please search for server actions with forms in next.js. It is nice to know how to handle it as it gives you the ability to have a server form component instead a client one